src/logic_utils/canonical_forms.h

```diff
@@ -0,0 +1,12 @@
+
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "truth_table_generator.cpp"
+#include "truth_table_generator.h"
+
+using namespace std;
+vector<Token> canonical_forms_sop(string expression);

```

Comment:
Please write all function names in `camelCase` instead of `snake_case`, also remove the forms word as it's redundant (eg, `canonicalSoP`)